ConcernsCaseWork/Service.TRAMS/NtiWarningLetter/NtiWarningLetterDto.cs

```diff
@@ -8,13 +8,15 @@ public class NtiWarningLetterDto
 	{
 		public long Id { get; set; }
 		public long CaseUrn { get; set; }
-		public DateTime CreatedAt { get; set; }
-		public DateTime UpdatedAt { get; set; }
+		public DateTime? DateLetterSent { get; set; }
 		public string Notes { get; set; }
+		public int? StatusId { get; set; }
```

Comment:
StatusId is nullable - is it valid to have a null status? Feels like the sort of thing that should have a default value if anything

Reply:
The status is not required according to requirements, and the default is the lack of it.